.github/workflows/lint.yml

```
@@ -10,7 +10,7 @@ on:
 jobs:
   golangci:
     name: golangci-lint
-    runs-on: ubuntu-latest
+    runs-on: buildjet-4vcpu-ubuntu-2004
```

```suggestion
    runs-on: ubuntu-latest
```

Copy/Paste from https://github.com/cosmos/cosmos-sdk/pull/12219#discussion_r895225390:

I don't think speeding-up this one is necessary either. It is running pretty fast too (cf https://github.com/cosmos/cosmos-sdk/actions/workflows/lint.yml)

---

Thanks Julien!